frontend/src/components/hub/HubSupporters.tsx

```diff
@@ -0,0 +1,226 @@
+      background: "white",
+    },
+    "& li button": {
+      width: "7px",
+      height: "7px",
+      border: "none",
+      background: "#66BCB5",
+    },
+  },
+  itemContainer: {
+    display: "flex",
+    alignItems: "center",
+    gap: "15px",
+  },
+  supporterImg: {
+    borderRadius: "50%",
+  },
+  supporterName: (containerClass) => ({
+    fontSize: "17px",
+    fontWeight: "600",
+    whiteSpace: "nowrap",
+    width: containerClass ? "170px" : "200px",
+    overflow: "hidden",
+    textOverflow: "ellipsis",
+    margin: 0,
+  }),
+  supporterSubtitle: (containerClass) => ({
+    margin: 0,
+    fontSize: "12px",
+    color: "#484848",
+    whiteSpace: "nowrap",
+    width: containerClass ? "170px" : "200px",
+    overflow: "hidden",
+    textOverflow: "ellipsis",
+  }),
+  carouselEntry: {
+    padding: " 8px",
+    display: "flex",
+    justifyContent: "left",
+    [theme.breakpoints.down("md")]: {
+      padding: 0,
+    },
+  },
+  containerInSmallDevices: {
+    display: "flex",
+    gap: "20px",
+    alignItems: "center",
+    backgroundColor: "#EEEFEEE8",
+    borderRadius: "4px",
+    padding: "10px",
+    marginBottom: theme.spacing(3),
+  },
+  supporterImgSmallDevice: {
+    borderRadius: "50%",
+  },
+  textAlign: {
+    marginLeft: "auto",
+  },
+  allSupporters: {
+    display: "flex",
+    alignItems: "center",
+    color: "#484848",
+    fontWeight: "600",
+    fontSize: "17px",
+    [theme.breakpoints.down("sm")]: {
+      fontSize: "15px",
+    },
+  },
+  arrowIcon: {
+    color: "#207178",
+  },
+}));
+
+const HubSupporters = ({ supportersList, containerClass }: HubSupporter) => {
+  const classes = useStyles({ containerClass: containerClass });
+  const isSmallOrMediumScreen = useMediaQuery<Theme>((theme) => theme.breakpoints.down("lg"));
+  const { locale } = useContext(UserContext);
+  const texts = getTexts({ page: "hub", locale: locale });
+  return (
+    <>
+      {!isSmallOrMediumScreen ? (
+        <HubSupportersSlider
+          classes={classes}
+          texts={texts}
+          containerClass={containerClass}
+          supportersList={supportersList}
+        />
+      ) : (
+        <HubSupportersInSmallDevice
+          classes={classes}
+          containerClass={containerClass}
+          supportersList={supportersList}
+          texts={texts}
+        />
+      )}
+    </>
+  );
+};
+
+export default HubSupporters;
+
+const CarouselItem = ({ supporter, classes, texts }) => {
+  return (
+    <div className={classes.carouselEntry} key={supporter.name}>
+      <div className={classes.itemContainer}>
+        <img
+          src={getImageUrl(supporter?.logo)}
+          width={76}
+          height={76}
+          alt={supporter.name}
+          className={classes.supporterImg}
+        />
+        <div>
+          <p className={classes.supporterName}>{supporter?.name}</p>
+          <p className={classes.supporterSubtitle}>
+            {texts.supporter}: {supporter.subtitle}
```

Comment:
```suggestion
            {supporter.subtitle}
```
I forgot to request this change in my first review. The "Supporter:" should not be hardcoded. The text shown here should be completely retrieved from the database to give us the maximum flexibility. Please remove this from the text file as well